### ISSUE TITLE
Building newer version in C++

### ISSUE THREAD

**Issue by steffenger** [sys-bio/roadrunner#1099]
Hello all,

I try to build a newer version of roadrunner in c++ with the usual commands from [docs](https://libroadrunner.readthedocs.io/en/latest/Installation/installation.html).

The build was succesfull but i compared it with a previous installation and i noticed an import difference in the lib/cmake installation folders. 

old installation:

![e84d2829-6bff-4e6d-b901-b47df3112eec](https://user-images.githubusercontent.com/95339856/235871213-c8aeaa6d-08c3-4738-9a52-3561803b4f95.PNG)

new installation:

![image](https://user-images.githubusercontent.com/95339856/235872060-baf6da4b-9c6d-4dac-8be5-4281dee61c27.png)

I compared the cmake files and the cmake configuration for both and didn't really find any differences. Do any of you have an idea or can someone recreate the error?

best regards 

steffen

**Comment by luciansmith** [comment]
It actually looks like nothing is installing in the 'lib' directory at all, unless that part was cut off of your screenshot?

I did try to recreate your issue, but was unable to--I got normal lib/ and lib/cmake/ directories.

I would recommend deleting your build and install directories, and starting from scratch.  I have seen odd things like this happen as a result of leftover files from previous builds in the past, so that would be my first guess.
